fischkapp/src/components/AddNewCard/AddNewFrontCard.tsx

```diff
@@ -6,14 +6,26 @@ import { StyledButtonsContainer } from "../styles/AddNewCard.styled";
 
 type FrontCardProps = {
   onFlipCard: Function;
+  onSetNewCard: Function;
```

Comment:
**nitpick**: These functions also should be typed more specifically. Now it's not using full Typescript power. Im leaving you an example below (in next comment)